web/scripts/generateZonesConfig.ts

```diff
@@ -24,7 +24,9 @@ const mergeZones = (): ZonesConfig => {
   );
 
   const zoneFiles = fs.readdirSync(basePath);
-  const filesWithDirectory = zoneFiles.map((file) => `${basePath}/${file}`);
+  const filesWithDirectory = zoneFiles
+    .filter((file) => file.endsWith('.yaml'))
```

Comment:
safety first :)

Reply:
I was very confused why it didn't work, but turns out the `.ds_store` file was throwing everything off 😬